tests/main/econnreset/task.yaml

```diff
@@ -38,7 +38,7 @@ execute: |
     iptables -I INPUT -p tcp --match hashlimit --hashlimit-mode srcip,dstip,srcport,dstport --hashlimit-above 512kb/s \
              --hashlimit-name 'econnreset' -j DROP
 
-    su -c "/usr/bin/env SNAPD_DEBUG=1 SNAPD_DEBUG_HTTP=3 snap download --edge test-snapd-huge 2>snap-download.log" test &
```

Comment:
Nothing wrong with this change afacit, but I wonder what was the motivation for it?

Reply:
Ah, ignore me, I see similiar question from zyga above.